ovmi/lib/src/predicates/is_less.rs

```diff
@@ -0,0 +1,51 @@
+use crate::executor::*;
+use crate::predicates::*;
+use crate::Range;
+
+pub struct IsLessThanPredicate<'a, Ext: ExternalCall> {
+    pub ext: &'a Ext,
```

Comment:
What's about to use `Arc` here instead of a specified lifetime? In my view `'a` makes interface  a bit dirty.